server/chat-plugins/scavenger-games.ts

```diff
@@ -355,15 +355,16 @@ const TWISTS: {[k: string]: Twist} = {
 
 		onComplete(player, time, blitz) {
 			const now = Date.now();
-			const takenTime = Chat.toDurationString(now - this.startTimes[player.id], {hhmmss: true});
-			const result = {name: player.name, id: player.id, time: takenTime, blitz};
+			const takenTime = now - this.startTimes[player.id];
+			const formattedTakenTime = Chat.toDurationString(takenTime, {hhmmss: true});
+			const result = {name: player.name, id: player.id, time: formattedTakenTime, duration: takenTime, blitz};
```

Comment:
```suggestion
			const formattedTakenTime = Chat.toDurationString(takenTime, {hhmmss: true});
			const result = {
						name: player.name,
						id: player.id,
						time: Chat.toDurationString(takenTime, {hhmmss: true}),
						duration: takenTime,
						blitz,
			};
```

Reply:
Think something went wrong here (formattedTakenTime is still shown); will push locally (thank)